Sources/PointFree/Atom.swift

```diff
@@ -34,8 +34,15 @@ public let atomLayout = View<AtomFeed> { atomFeed -> [Node] in
       [xmlns("http://www.w3.org/2005/Atom")],
       [
         title(atomFeed.title),
-        link([href(atomFeed.atomUrl), rel(.self)]),
-        link([href(atomFeed.siteUrl)]),
+        element(
+          "link",
+          [
+            .init("href", atomFeed.atomUrl) as Attribute<Void>,
+            .init("rel", "self")
+          ],
+          [""]
```

Comment:
this is the hack. I'm putting in a child node so that the closing tag is rendered :/

Reply:
Ha, no NB?

Reply:
done